src/Prima/Pyxis/Form.elm

```diff
@@ -1092,6 +1119,7 @@ renderInput state model ({ reader, slug, label, attrs } as config) validations =
             , ( "is-invalid", isFormSubmitted state && not valid )
             , ( "is-pristine", pristine )
             , ( "is-touched", not pristine )
+            , ( "has-warning", not warning )
```

Comment:
Sbaglio io o la logica è invertita? Se invece è corretta allora signfica che `warning` non è una variabile parlante.

Reply:
si, è un refuso, inizialmente la logica l'avevo pensata al contrario; devo cambiare il nome della variabile. La chiamerei noWarning

Reply:
secondo me è meglio mantenere i nomi e i concetti sempre al positivo. E' più facile pensare a `hasWarning` che poi può diventare facilmente `not << hasWarning`

Reply:
ok posso cambiare la funzione in "hasWarning"